sound/soc/sof/topology.c

```diff
@@ -2454,8 +2454,7 @@ static int sof_widget_ready(struct snd_soc_component *scomp, int index,
 	case snd_soc_dapm_dai_link:
 	case snd_soc_dapm_kcontrol:
```

Comment:
what about the above *known* widgets that we don't support? Is it still ok to not report them in the normal case, when debug printk is disabled?

Reply:
@lyakh yes, I think that just means that SOF doesnt currently support those known types. Handling of widgets in the driver is only optional. None of them are mandatory.